be/src/runtime/routine_load/data_consumer_group.cpp

```diff
@@ -171,8 +170,8 @@ Status KafkaDataConsumerGroup::start_all(StreamLoadContext* ctx) {
             VLOG(3) << "get kafka message"
                     << ", partition: " << msg->partition() << ", offset: " << msg->offset() << ", len: " << msg->len();
 
-            (kafka_pipe.get()->*append_data)(static_cast<const char*>(msg->payload()), static_cast<size_t>(msg->len()),
-                                             row_delimiter);
+            Status st = (kafka_pipe.get()->*append_data)(static_cast<const char*>(msg->payload()),
```

Comment:
Maybe it's better to add log of the `st` detail at line 183.

Reply:
Error message added.